lib/v2/jewishmuseum/exhibitions.js

```diff
@@ -0,0 +1,16 @@
+const buildData = require('@/utils/common-config');
+
+module.exports = async (ctx) => {
+    const link = `https://thejewishmuseum.org/exhibitions`;
+
+    ctx.state.data = await buildData({
+        link,
+        url: link,
+        title: `Jewish Museums - Exhibitions`,
+        item: {
+            item: 'article',
+            title: `$('article.exhibition h3').text()`,
+            link: `$('article.exhibition > a').attr('href')`,
+        },
+    });
+};
```

Comment:
There's one empty item in `/jewishmuseum/exhibitions`

<details>
<summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/11386903/161429849-b9c16e9f-d13e-4807-81dc-8bf38c9b9783.png)
</details>